woocommerce-services.php

```diff
@@ -1205,10 +1205,27 @@ public function add_meta_boxes() {
 			if ( $this->shipping_label->should_show_meta_box() ) {
 				add_meta_box( 'woocommerce-order-label', __( 'Shipping Label', 'woocommerce-services' ), array( $this->shipping_label, 'meta_box' ), null, 'side', 'default' );
 			}
+
+			add_meta_box( 'woocommerce-services-shipping-debug', __( 'Shipping Debug', 'woocommerce-services' ), array( $this, 'shipping_rate_packaging_debug_log_meta_box' ), 'shop_order', 'normal', 'default' );
+		}
+
+		public function shipping_rate_packaging_debug_log_meta_box( $post ) {
```

Comment:
What happens if the shipping log is non-existant? We should display a message explaining this to the user.

Reply:
What do you think about hiding the metabox instead?

Reply:
@jeffstieler - when the log is nonexistent, can we either remove the debug meta box or add an explanation why it's empty?